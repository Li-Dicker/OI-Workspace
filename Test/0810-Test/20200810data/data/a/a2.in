10
10
32478 76904 54710 11873 18866 81081 37326 79105 11782 72067 
37092 22169 44985 6149 46904 44621 22395 49772 30808 37162 
10
99951 54207 5380 75001 4616 22277 3267 76705 21811 41776 
9496 9252 7727 14061 19038 5879 34150 18639 44811 19438 
10
40037 8312 71610 77714 99516 21334 44636 86034 40423 94737 
20390 23106 28841 29625 30649 18769 30835 18936 18733 24741 
9
60054 7053 84991 1248 35997 87468 9145 27111 8239 
30229 20082 20284 13702 37190 776 5138 6003 29500 
10
24729 57835 60458 56437 50596 81280 34063 95789 27901 14286 
19115 28159 27307 29881 41145 7508 15587 22262 5715 21520 
9
40477 67399 66873 24254 31878 52997 86788 89745 29144 
44994 9734 18106 14017 27728 20453 6444 3239 36471 
10
72183 59735 83823 95338 42829 91436 95186 35896 45262 48422 
44538 30369 9666 8743 30091 1379 5546 5402 40819 19160 
9
74463 51659 60821 51293 87991 96639 59155 67501 1368 
27286 48718 44322 11570 37650 7682 41141 27554 7175 
9
20944 68527 96840 50159 29257 15965 22822 38201 67630 
196 28510 23924 22793 23535 11809 17158 37589 48110 
10
78009 33012 25177 61069 61729 76339 431 83265 24463 93582 
24559 26228 25156 26217 16896 48868 48606 44640 49168 11005 
