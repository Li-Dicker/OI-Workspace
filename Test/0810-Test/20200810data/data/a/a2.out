Case #1: 721266
Case #2: 548061
Case #3: 802580
Case #4: 402545
Case #5: 679573
Case #6: 592515
Case #7: 767278
Case #8: 739702
Case #9: 559942
Case #10: 833860
